src/lib/SendbirdSdkContext.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+// import { SendBirdState } from './SendbirdState';
 
 type ContextAwareComponentType = {
   (props: any): JSX.Element;
   displayName: string;
 };
 
+// <SendBirdState>
```

Comment:
what is this?

Reply:
Actually, it had to be
```tsx
export const SendbirdSdkContext = React.createContext<SendBirdState>({});
```

I took a pretty long time to find out the interface name,,, it will be easier to search what type the SendbirdSdkContext has with this keyword